.github/workflows/ci.yml

```diff
@@ -122,13 +122,13 @@ jobs:
     - name: Show byte order for debugging
       run: ${{ env.CARGO }} test --verbose $TARGET byte_order -- --nocapture
     - name: Run tests
-      run: ${{ env.CARGO }} test --verbose
+      run: ${{ env.CARGO }} test --verbose $TARGET
```

Comment:
Was this missed before this?

Reply:
Possibly, It's just a diagnostic and it's rarely anything other than little endian, so definitely easy to miss.